src/ikernel.ts

```diff
@@ -15,6 +15,10 @@ import {
 } from 'jupyter-js-utils';
 
 
+export
+type hist_access_type = 'range' | 'tail' | 'search';
```

Comment:
You'll need to update the required Typescript and use this version of TypeDoc: "https://github.com/SierraSoftworks/typedoc.git#cb5aea2b218b4f773451e2818a48629ee9c5066e"